src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.cs

```diff
@@ -209,7 +209,14 @@ public override void VisitLocal(ILocalSymbol symbol)
                 AddSpace();
             }
 
-            builder.Add(CreatePart(SymbolDisplayPartKind.LocalName, symbol, symbol.Name));
+            if (symbol.IsConst)
+            {
+                builder.Add(CreatePart(SymbolDisplayPartKind.ConstantName, symbol, symbol.Name));
```

Comment:
it's worth documenting the resoltion here.  or, at least, on the enum member itself.